### DISCUSSION TITLE
React Native Firebase Dynamic Links

### DISCUSSION THREAD

**Discussion by mohamed000123** [invertase/react-native-firebase#6601] [Q&A]
I am using react native firebase dynamic links, after completing my setup, i tried to use it, but i faced this error .

APP ERR Error: You attempted to use a firebase module that's not installed natively on your iOS project by calling firebase.dynamicLinks(). Ensure you have either linked the module or added it to your projects Podfile.

I tried pod install but still have the error

**Comment by mikehardy** [comment]
You have not correctly installed or built the app, neither have you included enough information to help.

Here's a script that works.
https://github.com/mikehardy/rnfbdemo/blob/main/make-demo.sh

**Reply by mohamed000123** [reply]
i did exactly like the docs step by step
i checked my setup many times i really don't know what is the wrong
